src/app/state/rest/index.ts

```diff
@@ -92,10 +92,36 @@ export const useCaseTypes = (options?: Options) => {
   })
 }
 
-export const useBAG = (bagId: string, options?: Options) => {
+/**
+ export const usePermitCheckmarks = (bagId: string) => {
+  const handleError = useErrorHandler()
+  const queryString = qs.stringify({ bagId }, { addQueryPrefix: true })
+  return useApiRequest<{ has_b_and_b_permit: boolean, has_vacation_rental_permit: boolean }>({
+    url: makeGatewayUrl("permits", "checkmarks", queryString),
+    groupName: "permits",
+    handleError,
+    getHeaders
+  })
+} 
+ */
```

Comment:
Should this be commented out?

Reply:
I copied it as example code, removed it now